Testing/Test-Validation.ps1

```diff
@@ -36,7 +36,7 @@ $webApp = Get-AzWebApp `
 
 $address = ./Deployment/Get-FunctionUri.ps1 `
     -WebApp $webApp `
-    -FunctionName 'RepositoryValidator'
+    -FunctionName 'StatusCheck'
```

Comment:
We should be testing the actual validation here, not the status. I think we could have a separate script, something like `Get-RepositoryValidatorStatus` or similar.

Reply:
whoops, I just used this for testing